docs/plugins/fpscounter.md

```diff
@@ -1,21 +1,20 @@
-# FPS counter
+# FPS Counter
 
-Frames per second (FPS) is an important measure for the performance of your App. A higher FPS indicates a smoother experience.
-When validating an App on a certain platform it's useful to get an indication of the FPS score.
+The number of *Frames per second (FPS)* is an important measure for the performance of your App. The higher the FPS, the smoother the experience. When you are validating your App on a certain platform, it's useful to get an indication of the *FPS score*.
 
-The SDK contains a built-in **FPS counter** that can be turned on or off via the [Platform Setting](/plugins/settings?id=platform) `showFps` in `settings.json`.
+The SDK contains a built-in *FPS counter* that can be enabled or disabled via the Platform Setting `showFps` in `settings.json`. When it is enabled, a *real-time* counter with the current FPS will be displayed as an overlay in the top left corner of your App.
```

Comment:
We used to have a link to the platform settings. Might make sense to keep that one ..